zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/system/BrokerAdminServiceTest.java

```diff
@@ -84,10 +84,31 @@ void shouldResumeStreamProcessorWhenRequested() {
     // given
     partitions.pauseProcessing();
 
+    final Future<?> response1;
+    try (final var client = zeebe.newClientBuilder().build()) {
+      response1 =
+          client.newPublishMessageCommand().messageName("test1").correlationKey("test-key").send();
+
+      // then
+      assertThat(response1)
+          .failsWithin(Duration.ofSeconds(5))
+          .withThrowableThat()
+          .havingCause()
+          .withMessageContaining("UNAVAILABLE: Processing paused for partition");
+    }
+
```

Comment:
:x: This can be removed from this as this test is verifying resume. So no need to assert behavior of pause.